src/constants/make.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
   PLYMOUTH: 'Plymouth',
   PONTIAC: 'Pontiac',
   PORSCHE: 'Porsche',
+  POLESTAR: 'Polestar',
```

Comment:
I'm not sure what sorting you're going for here, but I would probably move this above "Pontiac".